requirements.txt

```diff
@@ -1,4 +1,4 @@
 numpy
 ounce>=1.1.0,<2.0
-polliwog==1.0.0b6
+-e git://github.com/lace/polliwog.git@b18ab1c07c9be3b9762ae105488a8800d0547c41#egg=polliwog
```

Comment:
This isn't intended to be merged; it should be replaced with a new release of polliwog.

Reply:
Understood